fvid/fvid.py

```diff
@@ -171,7 +171,7 @@ def make_video(output_filepath, image_sequence):
         ffmpeg.input(
             f"{FRAMES_DIR}encoded_frames*.png",
             pattern_type="glob",
-            framerate="1/5",
+            framerate=framerate,
         ).output(output_filepath, vcodec="libx264rgb").run(quiet=True)
```

Comment:
Just to point out, there was recently another merge that changes this so there is currently a conflict, the fix is simple, just replace this line with:

`         ).output(outputfile, vcodec="libx264rgb").run(quiet=True)`

Reply:
Should be fixed.